### DISCUSSION TITLE
How to get rid of obsolete HM System variables in HA?

### DISCUSSION THREAD

**Discussion by Bastian007** [SukramJ/aiohomematic#856] [Q&A] [answered]
Dear all,
I have moved all program logic from RaspberryMatic into Home Assistant gradually. 
Hence, some System variables in HM became obsolete and have been deleted. 
However, in Home Assistant those variables are still existing as deactivated entities.

Is there a way to get rid of this old not-existing stuff?

Best regards, Bastian

custom_homematic: 1.26.5
RaspberryMatic: 3.67.10.20230114
<img width="687" alt="2023-01-18 11_04_39-Settings" src="https://user-images.githubusercontent.com/12425311/213143216-e340c701-ba40-448e-978a-675917d4e18e.png">

**Comment by SukramJ** [comment, marked as answer]
Activate these Sysvars/Programs, restart HA and then you can delete them.

**Reply by Bastian007** [reply]
Thanks a lot for you help!
This approach works fine and I also got rid of some programs (Controls) which do not exist anymore.
